src/test/java/com/ea/restaurant/grpc/GrpcOauth2ServiceImplE2eTest.java

```diff
@@ -0,0 +1,73 @@
+package com.ea.restaurant.grpc;
+
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.exceptions.EntityNotFoundException;
+import com.ea.restaurant.repository.AppClientRepository;
+import com.ea.restaurant.repository.AppClientScopeRepository;
+import com.ea.restaurant.service.Oauth2Service;
+import com.ea.restaurant.test.fixture.Oauth2Fixture;
+import com.ea.restaurant.test.util.AuthenticationCallCredentials;
+import net.devh.boot.grpc.client.autoconfigure.GrpcClientAutoConfiguration;
+import net.devh.boot.grpc.client.inject.GrpcClient;
+import net.devh.boot.grpc.server.autoconfigure.GrpcServerAutoConfiguration;
+import net.devh.boot.grpc.server.autoconfigure.GrpcServerFactoryAutoConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@SpringBootTest(
```

Comment:
Create a properties same ss you did on your code base

Reply:
done.